### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by yangfl**

### CODE REVIEW

File: samba/edit_fperm.cgi [hunk 1]
```diff
@@ -31,7 +31,7 @@ print &ui_table_row($text{'fperm_dirmode'},
 	&ui_textbox("directory_mode", &getval("directory mode"), 5));
 
 print &ui_table_row($text{'fperm_notlist'},
-	&ui_textbox("dont_descend", &getval("dont descend"), 40));
+	&ui_textbox("don't_descend", &getval("dont descend"), 40));
```
Comment (jcameron) [inline samba/edit_fperm.cgi]: This is a field name, so you shouldn't fix this typo

**Comment by jcameron** [comment]
Thanks - this looks good, except for that one comment I made on the "don't" field

**Comment by yangfl** [comment]
ok, i'll fix them when i get touch with my laptop.

**Comment by yangfl** [comment]
@jcameron OK, please have a look.

**Comment by jcameron** [comment]
Thanks for the fixes!

**Pull request merged**